KlaviyoSwift.podspec

```diff
@@ -1,10 +1,10 @@
 Pod::Spec.new do |s|
   s.name             = "KlaviyoSwift"
   s.version          = "2.0.1"
```

Comment:
Although there is no change in the SDK itself should we update the minor version for the rich push release as it's not really a patch since it's a new feature but at the same time it's not breaking change.

Reply:
yes

Reply:
If there's one place I'd lean toward syncing version numbers, its the test apps with their accompanying SDK version

Reply:
of course we still can't be perfect, like if we overhauled the UI that'd warrant a version increment, even if the SDK didn't change, so its not a perfect science

Reply:
wow ignore all of that, thought i was looking at the test app PR 😮‍💨

Reply:
Realizing now we've talked about this since you wrote this comment. But yes I do think rich push should increment the minor version, as a new feature.

Reply:
Ignored. It's been a long day :)